xml/System.Runtime.CompilerServices/CallerArgumentExpressionAttribute.xml

```diff
@@ -23,7 +23,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Allows capturing of the expressions passed to a method. NOTE: Is not yet implemented in C# compiler and does nothing.</summary>
+    <summary>Indicates that a parameter captures the expression passed for another parameter as a string. NOTE: This attribute is not yet implemented in the C# compiler and does nothing.</summary>
```

Comment:
NIT: Isn't the `NOTE:` part should go into standard `> [!NOTE]` section? But not sure if the inline one could get more attention, up to you, I am OK as is